bio_hansel/qc/checks.py

```diff
@@ -41,16 +41,56 @@ def is_missing_tiles(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Op
     Returns:
         None, None if less missing tiles than tolerate; otherwise, "FAIL", error message
     """
-    if not st.are_subtypes_consistent:
-        return None, None
+    error_status = None
+    error_messages = None
+
+    if st.are_subtypes_consistent:
+        error_status, error_messages = check_for_missing_tiles(is_fastq=st.is_fastq_input(),
+                                                               curr_subtype=st.subtype,
+                                                               df=df,
+                                                               exp=int(st.n_tiles_matching_all_expected),
+                                                               obs=int(st.n_tiles_matching_all),
+                                                               p=p)
+    else:
+        error_status = []
+        error_messages = []
+
+        subtype_list = st.subtype.split(';')
```

Comment:
I'm wondering if the type of `subtype` and the other fields that can have `;`-delimited values should be either string/int/float or list of whatever type. Rather than a string delimited list. We can convert to `;` lists when writing the outputs. What do you think?

Reply:
If we had them all as lists, a lot of the string joining will be alleviated thus cleaning up the code. I feel it would also make scenarios like "What if we had more than 1 entry for a result", easier to handle. 

I think if we did go ahead and change the variables to lists, it would require a lot of refactoring, and perhaps when we decide to nuke jellyfish or blast it would be a great idea to replace with lists then.

Reply:
Yeah, I'd be in favour of all those things in the next merge!